kimenyikevin/FreeMentor
# #168189753  Build documentation of API Endpoint with Swagger

#### What does this PR do?
Documenting api for all Endpoint
#### Description of Task to be completed?
have the following endpoint work 
- POST /api/v1/auth/signup
- POST /api/v1/auth/signin
- PATCH /api/v1/auth/user/:id
- GET /api/v1/auth/mentors
- GET /api/v1/auth/mentors/:id
- POST /api/v1/auth/sessions
- PATCH  /api/v1/auth/sessions/:sessionsId/reject
- PATCH  /api/v1/auth/sessions/:sessionsId/accept
#### How should this be manually tested?
After cloning repo, cd into it in your terminal
- make sure you install nodejs
- At this time this can be tested locally by run http://localhost:3000/api-docs into a browser
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#168189753
#### Screenshots (if appropriate)
#### Questions:
